Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -0,0 +1,158 @@
+//swiftlint:disable todo
+
+import XCTest
+
+class UserFeedbackUITests: BaseUITest {
+    override var automaticallyLaunchAndTerminateApp: Bool { false }
+    
+    override func setUp() {
+        super.setUp()
+        app.launchArguments.append(contentsOf: [
+            "--io.sentry.iOS-Swift.auto-inject-user-feedback-widget",
+            "--io.sentry.iOS-Swift.user-feedback.all-defaults"
+        ])
+        launchApp()
+    }
+    
+    func testSubmitFullyFilledForm() throws {
+        let widgetButton: XCUIElement = app.staticTexts["Report a Bug"]
```

Comment:
`m`: Is there any reason we can't use accessibility identifiers to find the button? These usually don't change that often, and they also work when you add localization, so it makes the test less brittle.

Reply:
fair